### DISCUSSION TITLE
Simple arrows for managing collection / item order and nesting

### DISCUSSION THREAD

**Discussion by elliottloxley** [directus/directus#18372] [Feature Requests]
### Summary

My proposal would be to have 4 arrows (2 if nesting is not available) on items and collections, these arrows would move the items both up and down and in and out of nesting. This would allow drag and drop to be used for quick and easy reordering, and explicit arrows for fine grain control.

Forgive me if this feature already exists in some form but I have not found it.

### Basic Example

See below a quick mockup of the proposal, this is quite a common design pattern in other apps and I think it would benefit Directus greatly. 
Obviously there is great room for improvement from my crude attempt design wise!

![directus-example](https://user-images.githubusercontent.com/28651797/235384909-408af6a0-6786-4139-b6ff-cc7133d2cb59.png)

### Motivation

I often have issues with the drag and drop functionality of Directus when rearranging collections and items. It is good for quickly ordering lots of items but sometimes lacks fine grain control. I find this most apparent with nesting, where I'll find myself wiggling my mouse wildly to try and get the correct nesting and location, which can be quite frustrating.

### Detailed Design

As seen in the basic example, this is quite a simple idea and so there is not much to expand on here.

You can see the four arrows per item each mapping to a different vector of movement. Up and down arrows would control the vertical order of the items, ie. up would move the item above the item on top of it, putting it an order closer to the top. 

Left and right arrows would control the nesting of the item, right would make the item a child of the item above it, and left would remove the parenthood of the item above it.

As you can see from the example, arrows would be greyed out if at the maximums of their respective movement directions. eg. top of list, bottom of list, fully nested, and fully un-nested

### Must Have:
- A set of arrows per item to control order and nesting

### Should have
- Consistent design with the rest of the Directus UI
- Simple intuitive functionality for ease of use by users

### Drawbacks

One drawback is the consumption of a lot of visual space on the item preview, as you can see by the example, it takes up an icons' worth of horizontal space within the item. 
This could be mitigated by hiding the feature until necessary, wither via the display options menu on the right, or by enabling per item via the item context menu. It could also appear on hover.

### Alternatives

The alternative is simply keeping the current functionality. It is not broken in any way, and is usable.
The impact is the added frustration every time a new item is added or moved that requires nesting or complicated positioning

### Adoption Strategy

I imagine adopting the feature would cause no breaking changes, and would purely be a visual addition. Using the location arrows would modify the position/ nesting data in the exact same way as the current drag and drop feature.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
